defaults/main.yml

```diff
@@ -21,9 +21,8 @@ ovirt_engine_setup_firewall_manager: 'firewalld'
 
 # This option is suggested from oVirt Documentation
 # https://www.ovirt.org/documentation/install-guide/chap-Installing_oVirt/
-ovirt_engine_setup_update_all_packages: true
 ovirt_engine_setup_update_setup_packages: false
-ovirt_engine_setup_offline: false
+ovirt_engine_setup_offline: "{{ he_offline_deployment | default(false) }}"
```

Comment:
I think you can remove the "default(false)" since we have this in defaults/main.yml

Reply:
Then it would fail if someone would use the role by itself.